docs/content/en/docs/concepts/cluster-topologies.md

```diff
@@ -0,0 +1,82 @@
+description: >
+  Explanation of standalone vs. management/workload cluster topologies
+---
+
+For trying out EKS Anywhere or for times when a single cluster is needed, it is fine to create a _standalone cluster_ and run your workloads on it.
+However, if you plan to create multiple clusters for running Kubernetes workloads, we recommend you create a _management cluster_.
+Then use that management cluster to manage a set of workload clusters.
+
+This document describes those two different EKS Anywhere cluster topologies.
+
+## What is an EKS Anywhere management cluster?
+An EKS Anywhere management cluster is a long-lived, on-premises Kubernetes cluster that can create and manage a fleet of EKS Anywhere workload clusters.
+The workload clusters are where customers run their applications.
+The management cluster can only be created and managed by the Amazon CLI `eksctl`.
+
+The management cluster runs on customer hardware on-premises and it does not require any connectivity back to AWS to function.
+Customers are responsible for operating the management cluster including (but not limited to) patching, upgrading, scaling, and monitoring the cluster control plane and data plane.
+ 
+## What’s the difference between a management cluster and a standalone cluster?
+From a technical point of view, they are the same.
+Regardless of which deployment topology a customer chooses, you always start by creating a singleton, standalone cluster that’s capable of managing itself.
+This shows examples of separate, standalone clusters:
+
+![Standalone clusters self-manage and can run applications](/images/eks-a_cluster_standalone.png)
+
+Once a standalone cluster is created, you have an option to use it to use it as a management cluster to create separate workload cluster(s) under it, hence making this cluster a long-lived management cluster.
+You can only use `eksctl` to create or delete the management cluster and the singleton standalone cluster.
+This shows examples of a mamagement cluster that deploys and manages multiple workload clusters:
+
+![Management clusters can create and manage multiple workload clusters](/images/eks-a_cluster_management.png)
+
+## What’s the difference between a management cluster and a bootstrap cluster for EKS Anywhere?
+
+A management cluster is a long-lived entity you have to actively operate.
+The _bootstrap_ cluster is a temporary, short-lived kind cluster that is created on a separate [Administrative machine]({{< relref "clusterworkflow/#administrative-machine" >}}) to facilitate the creation of an initial standalone or management cluster.
+You do not need to interact or operate the `kind` cluster.
+The `kind` cluster will be automatically deleted by the end of the initial cluster creation.
+
+## When should I deploy a management cluster?
+If you want to run three or more EKS Anywhere clusters, we recommend them to choose a management/workload cluster deployment topology because of the advantages listed in the table below.
```

Comment:
"If you want to run three or more EKS Anywhere clusters, we recommend you to ..."

Reply:
Done.